samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/DocumentImportController.cs

```diff
@@ -110,9 +114,26 @@ private enum SupportedFileType
                     return this.BadRequest($"Unsupported file type: {fileType}");
             }
 
-            await this.ParseDocumentContentToMemoryAsync(kernel, fileContent, documentImportForm);
+            var memorySource = new MemorySource(
+                documentImportForm.ChatId.ToString(),
```

Comment:
Documents uploaded to the global scope will have an empty ChatID. Looks like you are not handling those documents in the `GetSourcesAsync` API.

Reply:
this is fine since `ChatId` defaults to `string.Empty` in this case and the files can be retrieved with `GetSourcesAsync(kernel, "")`, which we can say is a special case.

we currently don't have any UI/designs for showing this

Reply:
That's ok (but it's probably better to make that clear in the comments).

Reply:
In addition, global document will potentially be included in the bot responses. If we don't make it clear that the source tab doesn't display global documents, it will likely cause confusion someday.

Reply:
ah I see, yeah you're right in that it could be confusing. I think we could include some small text here that notes this, what do you think? I will ping Ruokan as well to get her suggestion

Reply:
I would recommend # 1 for simplicity. We can expand later if need be.

Reply:
agreed. Ruokan is creating a quick mockup for this and in the meantime I've added a column for what I think the text _could_ be
<img width="459" alt="image" src="https://github.com/microsoft/semantic-kernel/assets/52973358/997fe6a1-a1e1-40f7-840f-a4922f77c6fb">

Reply:
incorporating Ruokan's feedback, "Scope" has been changed to "Access" and also "All chats" to "Global" from the above screenshot.